frontend/src/app/AppRoutes.tsx

```diff
@@ -148,6 +147,7 @@ const AppRoutes: React.FC = () => {
             <Route path="/groupSettings" element={<GroupSettingsPage />} />
           </>
         )}
+        <Route path="/hardwareProfiles/*" element={<HardwareProfileRoutes />} />
```

Comment:
I'd like to find a way to improve this so that the Route isn't so exposed -- like hide it behind SSAR -- but this is not the place for it.

Reply:
Maybe we can follow with another jira

something that wraps HardwareProfileRoute,  ClusterSettingsPage, etc... like how we have a helper for kebabs, we would have some HOC or what have you that does the check and either passes you through, or shows some custom 401 page

Reply:
nvm looks like you did that already

Reply:
The problem is this page needs to chunk our stuff up -- I think it might need to be something we do on the HardwareProfileRoutes object... I could look into this... that might work.